nekton/dcm2nii.py

```diff
@@ -61,27 +69,91 @@ def check_slice_thickness_variable(all_dcm_paths: List[str]) -> bool:
         )
```

Comment:
Maybe you can use read_file and stop before pixel, it is a pydicom feature I guess, to just read the headers.